src/library.js

```diff
@@ -2416,6 +2416,9 @@ mergeInto(LibraryManager.library, {
     if (!warnOnce.shown) warnOnce.shown = {};
     if (!warnOnce.shown[text]) {
       warnOnce.shown[text] = 1;
+#if ENVIRONMENT_MAY_BE_NODE
+      if (ENVIRONMENT_IS_NODE) text = 'warning: ' + text;
+#endif
```

Comment:
why?

(doesn't `console.warn` on node show up as a clear warning already?)

Reply:
No, it just sends the text to stderr of the process.   I can split this change out.

Reply:
Oh, right, we use stderr directly due to some issues with printing from threads, IIRC...

Reply:
Even if we use `console.warning` under node.. that is indistinguishable from other writes to stderr. 

Split out.